Templates/Kotlin/Operation.stencil

```diff
@@ -13,7 +13,7 @@ import javax.annotation.Generated
 
 @Generated("{% if moduleName %}{{ moduleName }}{% else %}com.shopify.syrup{% endif %}")
 {% endif %}
-class {{ name }}({{ operation.variables|renderKotlinArguments }}): {% if isQuery %}Query{% else %}Mutation{% endif %}<{{ operation.name }}Response> {
+class {{ name }}({{ operation.variables|renderKotlinArguments }}): {% if isQuery %}Query{% else %}{{ operation.typeName }}{% endif %}<{{ operation.name }}Response> {
```

Comment:
can we drop the `if` logic here and just render `operation.typeName`?

Reply:
No, queries can have typeName `Query` or `QueryRoot`.

Reply:
My comments above address this use case as well.